packages/datagrid/src/components/DefaultHeaderRenderer/DefaultHeaderRenderer.test.js

```diff
@@ -42,6 +42,54 @@ describe('#DefaultHeaderGrid', () => {
 		expect(onKeyDown).toHaveBeenCalledWith(event, 'colId');
 	});
 
+	it('should render DefaultHeaderGrid with filter button on quality Bar', () => {
```

Comment:
you don't have to update this story...  nobody is using this component anymore (sadly)